notary-api/src/utils/wibson-lib/cryptography/encription.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Remove all this `eslint-disable` lines. You don't have to ignore them as long as we delete this intruder: https://github.com/wibsonorg/notary-sdk/blob/ns-swagger-input-validation/notary-api/src/utils/wibson-lib/package.json 🙂 ... After that, lint will work.